uaclient/version.py

```diff
@@ -15,7 +15,7 @@
 from uaclient.exceptions import ProcessExecutionError
 from uaclient.system import subp
 
-__VERSION__ = "32.3"
```

Comment:
@lucasmoura keep in mind that youmerged the changelog PR but are missing the version bump to 33.2